docs/src/modules/home/BaseUIThemesDemo.tsx

```diff
@@ -11,7 +11,7 @@ import { MenuButton } from '@base_ui/react/MenuButton';
 import { Modal, modalClasses } from '@base_ui/react/Modal';
 import { Option } from '@base_ui/react/Option';
 import { Select } from '@base_ui/react/Select';
-import { Slider, sliderClasses } from '@base_ui/react/Slider';
```

Comment:
This is the temperature slider on the `/base-ui` landing page, I will try to recreate it using the new slider